src/commands.cc

```diff
@@ -1098,16 +1099,12 @@ const CommandDesc echo_cmd = {
 
 KeymapMode parse_keymap_mode(const String& str)
 {
-    if (prefix_match("normal", str)) return KeymapMode::Normal;
```

Comment:
We are loosing that prefix behaviour here, but I think its actually better, we have completion for that.